### ISSUE TITLE
Annealing learning rate

### ISSUE THREAD

**Issue by SeanNaren** [SeanNaren/deepspeech.pytorch#26]

**Comment by SeanNaren** [comment]
Added [here](https://github.com/SeanNaren/deepspeech.pytorch/commit/4e048ba157f0f57ace3d6bfeb243c0335df8984d).
